coursier.bzl

```diff
@@ -180,7 +180,19 @@ def generate_imports(repository_ctx, dep_tree, srcs_dep_tree = None):
 
             target_import_string.append("".join(target_import_labels) + "\t],")
 
-            # 5. Finish the java_import rule.
+            # 5. Add a tag with the original maven coordinates for use generating pom files
```

Comment:
Please add a link to bazel_common to show how this is used, or https://github.com/google/bazel-common/blob/f1115e0f777f08c3cdb115526c4e663005bec69b/tools/maven/pom_file.bzl#L177

Reply:
Done